src/BootstrapReport/main.py

```diff
@@ -209,19 +209,19 @@ def pp_plot(self, confidence_band = True, alpha = 0.05, outfile = None, **kwargs
         plt.rc('legend', fontsize = plt_set['legend_fontsize'])
         plt.rc('axes', labelsize = plt_set['labelsize'])
         fig, ax = plt.subplots(figsize = (10, 10))
+        ax.set_ylim(0, 1)
+        ax.set_xlim(0, 1)
 
         if confidence_band == True:
             ax.fill_between(dkw_xgrid, dkw_lbound, dkw_ubound, color = plt_set['bandcolor'], label = 'Confidence band', alpha = 0.35)
         ax.scatter(replicates_eval_normcdf, replicate_ecdf, s = plt_set['pointsize'],
-                    c = plt_set['pointcolor'], label='Bootstrap replicates')
+                    c = plt_set['pointcolor'], label='Bootstrap reps.')
         ax.set_xlabel("CDF of normal distribution")
         ax.set_ylabel("CDF of bootstrap distribution")
         ax.legend(edgecolor = 'k', loc = 'upper left')
         ax.axline((0, 0), (1, 1), color="black", linestyle=(0, (5, 5)))
-        ax.text(0.52, 0.06, plot_data, fontsize = plt_set['legend_fontsize'], \
-            verticalalignment = 'bottom', horizontalalignment='left', bbox = props)
-        ax.set_ylim(0, 1)
-        ax.set_xlim(0, 1)
+        ax.text(0.955, 0.055, plot_data, fontsize = plt_set['legend_fontsize'], \
```

Comment:
@MosesStewart is it possible to left-justify the text in the lower right box? I think that was the previous default; now it seems to be right-justified.

Thanks!

![image](https://github.com/JMSLab/BootstrapReport/assets/7840353/f5918f72-9e87-4591-8c7e-88f8ef0822ba)

Reply:
@jmshapir the text gets justified to the same side as where we decide to place the box, which was why we had it the other way before. 

The only way I could think of accomplishing both left-justified text and a right-aligned text box would be to set it to left aligned, and have a series of `if` - `then` statements counting how many digits thee `num. replicates` are then changing the coordinates accordingly. If this sounds alright then I can implement.

Reply:
@MosesStewart thanks!

In that case it seems cleaner to leave it right-justified.

@rcalvo12 do you agree?

Reply:
@MosesStewart I think there might be a way to achieve what @jmshapir is looking for. See [this stack exchange thread](https://stackoverflow.com/questions/25771752/position-text-box-fixed-in-corner-and-correctly-aligned) which references [this function](https://matplotlib.org/stable/api/offsetbox_api.html) within matplotlib. However, if this proves tricky to implement, I agree with @jmshapir that leaving it as right justified would be fine.

Reply:
@rcalvo12 thank you for sharing! I had no idea that the class `offsetbox` existed within Matlab!

I implemented a working version below:

![ppplottest2](https://github.com/JMSLab/BootstrapReport/assets/111905359/a61daa33-749a-4624-99fa-8caf50345889)

There were some limitations, in that I could not adjust the transparency of the box, and I could not get the box to have rounded corners, so I changed the legend box to look the same as the `offsetbox`. If this looks okay then I can commit!

Reply:
@MosesStewart awesome, I think you can adjust some of those things by using a syntax like this:
```
        ob = offsetbox.AnchoredText(plot_data, loc='lower right')
        ob.patch.set(boxstyle = 'round, pad = 0.75, rounding_size = 0.3', facecolor = 'white', alpha = 0.86)
```

Reply:
@rcalvo12 thanks implemented in c8ce870

Reply:
![ppplottest2](https://github.com/JMSLab/BootstrapReport/assets/111905359/1279be36-9220-4e78-834b-991b0b521ba4)